SEP-0002-revised.md

```diff
@@ -0,0 +1,262 @@
+* nominate a Ombudsperson,
+* ensure that the governance of the project is running as intended.
+
+The Advisory Board is self-electing, with members appointed by a vote of the existing Advisory Board.
+
+#### Membership
+
+* The Advisory Board shall consist of at least 3 members and no more than 5 members. The Advisory Board selects how many members within this range are needed.
+* All members shall serve a maximum of a two-year term, and may be re-appointed at the end of their term.
+* Membership terms should be staggered so that approximately half the number of members will end their terms in any given year.
+* The Advisory Board may appoint members to its number up to the maximum number of members.
+* The Advisory Board should work to ensure that the composition of the Advisory Board represents a diverse cross-section of the scientific community with an interest in SunPy. This diversity should consider things such as research topic, career stage, as well as all other forms of diversity such as gender.
+* An Advisory Board member may be removed by a two-thirds majority vote of the Advisory Board or by Project Members following the same procedure as for the removal of a member of the Steering Committee.
+
+#### Meetings
+
+* The Advisory Board shall meet with the Steering Committee at least every quarter.
+* At least one week (7 days) notice of a meeting shall be provided to all eligible to attend.
+* Notice of meetings shall provide the information necessary to participate in the meeting.
+* Minutes for the meetings shall be recorded and made public as much as is appropriate.
+
+The Advisory Board shall organize itself as it sees fit.
+
+#### Advise and Consent
+
+The Steering Committee shall present the state of the project and future plans at meetings of the Advisory Board on a yearly basis.
+The Advisory Board shall advise and consent on those plans.
+
+#### Requiring a Project Vote
+
+In the unlikely event that the Advisory Board does not agree with a decision made by the Steering Committee, it can require Project Members to consent to the decision through a vote (see below for process).
+
+### The Steering Committee
+
+The role of the Steering Committee, which is chaired by the Lead Developer, is to lead the day-to-day operations of the project.
+The Steering Committee shall be responsible for the running of the project under advisement of the Advisory Board.
+This includes:
+
+* holding regular meetings with the developer community (at least monthly)
+* making regular reports to the Advisory Board (at least quarterly)
+* having ownership privileges to the SunPy GitHub organization and other community assets
+* creating, maintaining and enforcing the SunPy code of conduct (see below)
+* accepting or rejecting SEPs based on community consensus (excluding this SEP, see below)
+* delegating responsibility and authority to other community roles
+* planning and managing finances and financial processes of the project.
+
```

Comment:
Another responsibility of the Steering Committee is to enter into and enforce formal obligations with external parties (e.g. NumFocus). It is sort of implied in some of these bullet points, but I think it should be called out explicitly (somehow).

Reply:
```suggestion
* Ensuring SunPy adheres to its legal obligations, such as to its fiscal sponsor.

```